src/webextension/background/index.js

```diff
@@ -8,7 +8,10 @@ import initializeMessagePorts from "./message-ports";
 // XXX: For now, initialize the datastore on startup and then hook up the
 // button. Eventually, we'll have UX to create new datastores (and persist
 // existing ones).
-datastore.initialize().then(() => {
+datastore(async ds => {
```

Comment:
This is something we can discuss, but I generally prefer to put parens around the arg for single-arg arrow functions in JS; I think it's a more-consistent style, and probably slightly safer. `a => b` looks a lot like `a >= b`, even though they're *very* different.

I'm not married to that style, but I think it'd be good if we had a rule for it either way (which we could add to eslint).

Reply:
I changed them to all use parens.